### DISCUSSION TITLE
Dynamically Adding / Removing a Camera

### DISCUSSION THREAD

**Discussion by dsshap** [Motion-Project/motion#1475] [General]
Hi - I want to confirm that there is no way to add a new camera to be monitored without affecting the RTSP connections of other cameras defined in `./motion/conf.d/cam_x.conf`

Creating a new camera conf and issuing SIGHUP causes motion to disconnect and reconnect on all the RTSP connections. Which makes sense. 

I havent been able to find an option to tell motion to start / stop monitoring an individual camera.

**Comment by genius3000** [comment]
You are correct, only issuing a SIGHUP to essentially restart motion or actually restarting motion will add or remove a camera.

**Comment by angrycoding** [comment]
> Hi - I want to confirm that there is no way to add a new camera to be monitored without affecting the RTSP connections of other cameras defined in `./motion/conf.d/cam_x.conf`
> 
> Creating a new camera conf and issuing SIGHUP causes motion to disconnect and reconnect on all the RTSP connections. Which makes sense.
> 
> I havent been able to find an option to tell motion to start / stop monitoring an individual camera.

Actually there is a way to achieve this. Create for example 6 configuration files for cameras, and set netcam_url to empty value in each of the files. Start motion with those configuration files. Then use webcontrol url to dynamically set / clear netcam_url. That is the way I do that in my setup and it works quite well without affecting other cameras.
